mirage/seed_image/catalog_seed_image.py

```diff
@@ -119,6 +119,18 @@ def make_seed(self):
         self.output_dims = (self.nominal_dims * np.array([self.coord_adjust['y'],
                                                           self.coord_adjust['x']])).astype(np.int)
 
+        # change the values for the NIRISS/POM mode.  Add 137 pixels extra space around the main image area, full frame.
```

Comment:
I think it would be cleaner to take these lines out of the main function and place them into the `calcCoordAdjust` function, in a separate block for POM observations.

Also, you'll see in there that I use `self.grism_direct_factor`, which is defined  in the `__init__` function. I did this to make it easier on developers in case we ever want to change that factor for the grism. Feel free to set up the pom factor in the same way, or just within `calcCoordAdjust`.

Also, the grism_direct_factor at the moment is put into use if `grism_source_image` is true in the input yaml file. So this should work for NIRISS as well as NIRCam.